src/mudblazor/MudBlazor.Template/Components/Layout/NavMenu.razor

```diff
@@ -1,21 +1,57 @@
-﻿<MudNavMenu>
+﻿@*#if (IndividualLocalAuth)
+@implements IDisposable
+
+@inject NavigationManager NavigationManager
+
+##endif*@
+
+<MudNavMenu>
     <MudNavLink Href="" Match="NavLinkMatch.All" Icon="@Icons.Material.Filled.Home">Home</MudNavLink>
-@*#if (UseServer || UseWebAssembly)
+    @*#if (UseServer || UseWebAssembly)
     <MudNavLink Href="counter" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.Add">Counter</MudNavLink>
-##endif*@
+    ##endif*@
     <MudNavLink Href="weather" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.List">Weather</MudNavLink>
+    @*#if (IndividualLocalAuth)
+    <MudNavLink Href="auth" Match="NavLinkMatch.Prefix" Icon="@Icons.Material.Filled.List">Auth Required</MudNavLink>
```

Comment:
It looks like the Auth pages didn't exist in the older templates.  This was added by the .NET 8 template, and I converted code to MudBlazor.